{"text": "2024-05-10 06:24:57 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.133067", "seconds": 1.133067}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:24:57.539182+08:00", "timestamp": 1715293497.539182}}}
{"text": "2024-05-10 06:25:04 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:08.534671", "seconds": 8.534671}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:04.940786+08:00", "timestamp": 1715293504.940786}}}
{"text": "2024-05-10 06:25:07 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:11.062603", "seconds": 11.062603}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:07.468718+08:00", "timestamp": 1715293507.468718}}}
{"text": "2024-05-10 06:25:09 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode bytes in position 1-2: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode bytes in position 1-2: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:00:12.760870", "seconds": 12.76087}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode bytes in position 1-2: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode bytes in position 1-2: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:09.166985+08:00", "timestamp": 1715293509.166985}}}
{"text": "2024-05-10 06:25:11 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:15.000187", "seconds": 15.000187}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:11.406302+08:00", "timestamp": 1715293511.406302}}}
{"text": "2024-05-10 06:25:40 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:44.551030", "seconds": 44.55103}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:40.957145+08:00", "timestamp": 1715293540.957145}}}
{"text": "2024-05-10 06:25:40 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:44.551797", "seconds": 44.551797}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:25:40.957912+08:00", "timestamp": 1715293540.957912}}}
{"text": "2024-05-10 06:26:12 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:15.633285", "seconds": 75.633285}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:12.039400+08:00", "timestamp": 1715293572.0394}}}
{"text": "2024-05-10 06:26:14 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:17.978135", "seconds": 77.978135}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:14.384250+08:00", "timestamp": 1715293574.38425}}}
{"text": "2024-05-10 06:26:15 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x99 in position 1: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x99 in position 1: invalid start byte\n", "record": {"elapsed": {"repr": "0:01:19.563762", "seconds": 79.563762}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x99 in position 1: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x99 in position 1: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:15.969877+08:00", "timestamp": 1715293575.969877}}}
{"text": "2024-05-10 06:26:18 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:21.898970", "seconds": 81.89897}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:18.305085+08:00", "timestamp": 1715293578.305085}}}
{"text": "2024-05-10 06:26:47 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:51.449440", "seconds": 111.44944}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:47.855555+08:00", "timestamp": 1715293607.855555}}}
{"text": "2024-05-10 06:26:47 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:51.450051", "seconds": 111.450051}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:26:47.856166+08:00", "timestamp": 1715293607.856166}}}
{"text": "2024-05-10 06:27:18 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:22.545528", "seconds": 142.545528}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:18.951643+08:00", "timestamp": 1715293638.951643}}}
{"text": "2024-05-10 06:27:21 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:24.717233", "seconds": 144.717233}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:21.123348+08:00", "timestamp": 1715293641.123348}}}
{"text": "2024-05-10 06:27:22 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xd3 in position 3: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 3: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:02:26.243431", "seconds": 146.243431}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xd3 in position 3: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xd3 in position 3: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:22.649546+08:00", "timestamp": 1715293642.649546}}}
{"text": "2024-05-10 06:27:25 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:28.613553", "seconds": 148.613553}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:25.019668+08:00", "timestamp": 1715293645.019668}}}
{"text": "2024-05-10 06:27:54 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:02:58.163295", "seconds": 178.163295}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:54.569410+08:00", "timestamp": 1715293674.56941}}}
{"text": "2024-05-10 06:27:54 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:02:58.163995", "seconds": 178.163995}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:27:54.570110+08:00", "timestamp": 1715293674.57011}}}
{"text": "2024-05-10 06:28:25 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:29.262177", "seconds": 209.262177}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:28:25.668292+08:00", "timestamp": 1715293705.668292}}}
{"text": "2024-05-10 06:28:25 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3945), thread 'MainThread' (139869865110400): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic-action/epic-action/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic-action/epic-action/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:29.429182", "seconds": 209.429182}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-action/epic-action/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3945), thread 'MainThread' (139869865110400):", "module": "claim", "name": "__main__", "process": {"id": 3945, "name": "MainProcess"}, "thread": {"id": 139869865110400, "name": "MainThread"}, "time": {"repr": "2024-05-10 06:28:25.835297+08:00", "timestamp": 1715293705.835297}}}
